todo.md

```diff
@@ -0,0 +1,11 @@
+# TODOs for json schema generation
```

Comment:
I planned to remove as soon as everything is done. Any thoughts on the enum with bit_flag problem? What is expected to be in the json. The name of the enum (=string) or a number? String is only possilbe when one flag is true so I assume that a number should be the right thing?